### DISCUSSION TITLE
[I-PI Installation] dp_ipi  and dp_ipi_low executable libraries not getting  installed

### DISCUSSION THREAD

**Discussion by AnuragKr** [deepmodeling/deepmd-kit#2369] [Q&A] [answered]
Hi,

As I wanted to run an MD simulation using I-PI, so I was installing deepmd from the source code [link](https://docs.deepmodeling.com/projects/deepmd/en/master/install/install-from-source.html). 
After completing the whole process I am not getting the desired file output which is mentioned in the link at the end.
I should be getting --
1. dp_ipi  2. dp_ipi_low

But I am getting only the dp_gmx_patch executable library.

Please let me know if any extra flag needs to be set or something else I need to do for installing the dp_ipi and dp_ipi_low executable libraries.

**Comment by njzjz** [comment, marked as answer]
This is a bug and has been fixed in #2296.

**Reply by AnuragKr** [reply]
Thanks for the help.
